server/src/remote.rs

```diff
@@ -108,7 +110,6 @@ impl FlightService for FlightServiceHandler {
     ) -> Result<Response<FlightInfo>, Status> {
         Err(Status::unimplemented("Not yet implemented"))
     }
-
```

Comment:
There should be a newline between functions.

Reply:
Fixed